src/System.Reflection.Emit/tests/TypeBuilder/TypeBuilderCreateType.netstandard.cs

```diff
@@ -0,0 +1,89 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Reflection.Emit.Tests
+{
+    public class TypeBuilderCreateType
+    {
+        [Theory]
+        [InlineData(TypeAttributes.Abstract)]
+        [InlineData(TypeAttributes.AnsiClass)]
+        [InlineData(TypeAttributes.AutoClass)]
+        [InlineData(TypeAttributes.AutoLayout)]
+        [InlineData(TypeAttributes.BeforeFieldInit)]
+        [InlineData(TypeAttributes.Class)]
+        [InlineData(TypeAttributes.ClassSemanticsMask | TypeAttributes.Abstract)]
+        [InlineData(TypeAttributes.NotPublic)]
+        [InlineData(TypeAttributes.Public)]
+        [InlineData(TypeAttributes.Sealed)]
+        [InlineData(TypeAttributes.SequentialLayout)]
+        [InlineData(TypeAttributes.Serializable)]
+        [InlineData(TypeAttributes.SpecialName)]
+        [InlineData(TypeAttributes.StringFormatMask)]
+        [InlineData(TypeAttributes.UnicodeClass)]
+        public void CreateType(TypeAttributes attributes)
+        {
+            TypeBuilder type = Helpers.DynamicType(attributes);
+            Type createdType = type.CreateType();
+            Assert.Equal(type.Name, createdType.Name);
+
+            Assert.Equal(type.CreateTypeInfo(), type.CreateTypeInfo());
```

Comment:
Thanks for pointing that out. Fixed it with Assert.Equal(type.CreateTypeInfo(), createdType.GetTypeInfo());